### PR TITLE
Add testimonials single pattern

### LINKED ISSUE #9152: Create `Testimonials: single` pattern
Single

Check Figma designs F5AAbRJOGbTeEweXPFtHgf-fi-1695_33013 for the latest updates and mobile views.

### PR DESCRIPTION
**Pull request by roykho**
Fixes https://github.com/woocommerce/woocommerce-blocks/issues/9152

This PR adds the testimonials single pattern.

### Screenshots

| Proposed design | Actual display |
| ------ | ----- |
|        |       |

#### User Facing Testing

1. Create a page/post and use the block inserter to add patterns->WooCommerce->Testimonials Single
2. Ensure the display is similar to the proposed design in Editor and Frontend.

* [ ] Do not include in the Testing Notes 

### WooCommerce Visibility

* [x] WooCommerce Core
* [ ] Feature plugin
* [ ] Experimental

### Changelog

> Add testimonials single pattern

### CODE REVIEW

File: patterns/testimonials-single.php [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+		<!-- /wp:column -->
+
+		<!-- wp:column {"layout":{"type":"default"}} -->
+		<div class="wp-block-column"><!-- wp:heading {"level":4,"style":{"typography":{"textTransform":"none","fontSize":"18px"}}} -->
+		<h4 class="wp-block-heading" style="font-size:18px;text-transform:none"><strong>Great experience</strong></h4>
+		<!-- /wp:heading -->
+
+		<!-- wp:paragraph {"style":{"typography":{"fontSize":"18px"}}} -->
+		<p style="font-size:18px">
+					In the end the couch wasn't exactly what I was looking for but my experience with the
```
Comment (albarin) [inline patterns/testimonials-single.php]: When inserted in the editor, it looks like there's an extra space at the beginning
Screenshot 2023-04-25 at 10 58 21

**Review by albarin — APPROVED** [review]
👏 Left a small comment, but pre-approving.

**Comment by shaunandrews** [comment]
Looks good. My only comment is the name: "Testimonials Single" seems like an oxymoron. Shouldn't it be "Single Testimonial"?

**Comment by roykho** [comment]
So I am following the naming convention of existing testimonial titles we already have seen here https://github.com/woocommerce/woocommerce-blocks/blob/trunk/patterns/testimonials-3-columns.php#L3

This is so "testimonials" are of consistent titles. I believe the idea is `Testimonials` is like the pattern type if you will. I am happy to change it but we might need to change the other pattern we already as well to match.

**Pull request merged**